### ISSUE TITLE
Increase in body weight is lowered when mice received fecal microbiota transfer from donor mice treated with the AT1 receptor antagonist telmisartan

### ISSUE THREAD

**Issue by SvetlanaUP** [waldronlab/BugSigDBcuration#529]
Increase in body weight is lowered when mice received fecal microbiota transfer from donor mice treated with the AT1 receptor antagonist telmisartan – Marco L. Freschi – Frontiers in Pharmacology
https://www.frontiersin.org/journals/pharmacology/articles/10.3389/fphar.2024.1453989/full

**Comment by annemarie-sharp** [comment]
Hello @SvetlanaUP please can I curate this

**Comment by annemarie-sharp** [comment]
Thank you @SvetlanaUP

**Comment by annemarie-sharp** [comment]
Hello @SvetlanaUP this article is ready for review https://bugsigdb.org/39624839

**Comment by KateRasheed** [comment]
Good morning @SvetlanaUP . I'll like to review this paper please.

**Comment by KateRasheed** [comment]
Good morning @annemarie-sharp . Welldone on the curation of this paper. Here are things to note:

- Please add this to experiment 1 group 1 description: "after the mice had already been fed High Fat Diet (HFD) for 7 weeks "
- Remove the alpha diversity in experiment 1, see Figure 4A; the comparison was not done between d0 vs d49
- _Verrucomicrobiota_ and _Akkermansiaceae_ in experiment 1 are increased not decreased
- Replace _Patescibacteria group_ with [Candidatus Altimarinota](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Tree&id=363464&lvl=3&keep=1&srchmode=1&unlock) in experiment 1 signature 2; the latter belongs to the clade group while the former is the phylum
- Please curate the taxa in Figure S5B; some of them are not in Table S1. For example _Schaedlerella_, _UBA3282_
- For experiment 1 sample sizes, see this: "At days 0 and 49, the mice were in groups of 4 animals each in the cages."
- For experiment 2 sample sizes, please remove them; there was no clear figure and we don't want to assume. See this: "n = 11–12." 
- In experiment 2 signature 1, use this id (1262987) for CAG:495
- There was no antibiotics exclusion in this paper

Thank you so much.

**Comment by annemarie-sharp** [comment]
Good morning @KateRasheed  thank you so much, and thank you for your keen observation and corrections. Thanks for explaining that clade/phylum part. I am making the corrections now. 
Please I want to clarify, in our peer reviews, some info are not explicitly stated but explained in the article and I see they are figured out from the article and curated like that. So, I’m confused how best to go about similar situation in further curations, because info like the antibiotics exclusion and sample sizes were clearly explained in the article but not explicitly stated like sample size for d0 = 6
And please how can I curate Figure S5B and will it be in a different experiment since Cohens effect size was used?

**Comment by annemarie-sharp** [comment]
@KateRasheed Please one more question, in the article CAG-495 is genus but Proteobacteria bacterium CAG:495 is a species. I have observed that sometimes a different taxa level is curated if what is in the article isnt available on BugSigDB or NCBI. It confuses me also, please is there a guide to this?

**Comment by KateRasheed** [comment]
> Good morning [@KateRasheed](https://github.com/KateRasheed) thank you so much, and thank you for your keen observation and corrections. Thanks for explaining that clade/phylum part. I am making the corrections now. Please I want to clarify, in our peer reviews, some info are not explicitly stated but explained in the article and I see they are figured out from the article and curated like that. So, I’m confused how best to go about similar situation in further curations, because info like the antibiotics exclusion and sample sizes were clearly explained in the article but not explicitly stated like sample size for d0 = 6 And please how can I curate Figure S5B and will it be in a different experiment since Cohens effect size was used?

If it is not explicitly stated, we don't curate them because we don't want to assume.

For Fig. S5B, you just need to add it to experiment 1 because they are together; as seen in this excerpt: "Taxonomic changes could be observed depending on the HFD diet, as the abundances of phylum, family, and genus differed very clearly between d0, when the mice were still fed chow diet, and those at d49, after the mice had already been fed HFD for 7 weeks ([Supplementary Figures S3–S6](https://www.frontiersin.org/journals/pharmacology/articles/10.3389/fphar.2024.1453989/full#SM1))."

**Comment by KateRasheed** [comment]
> [@KateRasheed](https://github.com/KateRasheed) Please one more question, in the article CAG-495 is genus but Proteobacteria bacterium CAG:495 is a species. I have observed that sometimes a different taxa level is curated if what is in the article isnt available on BugSigDB or NCBI. It confuses me also, please is there a guide to this?

This happens sometimes. Best practice is to curate the same rank stated in the article. However, there are instances where a particular species in NCBI is assigned the genus rank in an article. **_Escherichia/Shigella sp._** as an example; we curate the way it's stated in NCBI not minding if some article assign it to the genus rank.

**Comment by annemarie-sharp** [comment]
Okay thank you @KateRasheed, duly noted. Thank you for explaining, I'll be done with the corrections shortly

**Comment by annemarie-sharp** [comment]
Hello @KateRasheed, I have made all other corrections but please I have not been able to get a clear understanding of how to curate figure S5B. For the points that fall on the line, do I curate them as increased/decreased or omit them?

**Comment by KateRasheed** [comment]
Good morning @annemarie-sharp . For the taxa that fall on the line, you can use the values for each group to determine which is increased or decreased. For example, d0 is 35.8 while d49 is 81.3 (check the yellow colored box) for _**Schaedlerella**_. This means _**Schaedlerella**_ is increased in d49.

Do let me know if this clear. Thank you so much.

**Comment by annemarie-sharp** [comment]
Good morning @KateRasheed Thank you for explaining that, it's clear. I have made the necessary changes now.

**Comment by KateRasheed** [comment]
Thanks for effecting these corrections @annemarie-sharp .

I added the sample size in experiment 1 and corrected 3 taxa with errors in signature 2 of this experiment.

This review is complete @SvetlanaUP

**Comment by SvetlanaUP** [comment]
https://bugsigdb.org/39624839 reviewed.

**Comment by annemarie-sharp** [comment]
Hello @KateRasheed thank you. Please the sample size for experiment1 is 6 not 4, that's what I tried explaining to you at first. It was explained in the article, there were 24 mice put into cages of four animals each, that is equal to 6cages. The article later stated that the samples were pooled from each cage, that is each of the six cages produced 1sample each, hence 6samples at day 0 and six samples at day 49.
The article also stated that for BL/6>fVEH and BL/6>fTEL, the samples were collected from each of the individual 24mice before killing them, hence sample sizes of 12 each for experiment2. 
Those are the sample sizes I initially used, as it was well explained in the article, please confirm.

**Comment by KateRasheed** [comment]
Hello @annemarie-sharp . This was my reason for stating four mice in experiment 1. "At days 0 and 49, the mice were in groups of 4 animals each in the cages. The stool samples were collected from the cages. Therefore, **one data point representing d0 and49 summarizes 4 mice**".

If this isn't clear, it's best we leave it blank. Usually when there is no consistent sample size, we leave it blank instead of assuming.

For experiment 2, the sample size was 11-12 as seen in the figure caption. Hence, the need to leave it blank.

**Comment by annemarie-sharp** [comment]
Hello @KateRasheed, I understand that's why I confirmed from you and later left it blank. But I feel putting sample size of 4 there will be misleading since the sample size is actually 6. I agree the article stated:
 '"At days 0 and 49, the mice were in groups of 4 animals each in the cages'  (24 animals in cages of 4animals each = 6cages)
 But it also went on to state
 'The samples at d0 and d49 were collected from the cages and thus represent the pooled microbiota of four experimental animals. The samples at the end of the experiment at d101 were taken directly from the colon at the time of killing and represent the specific microbiota of a single animal.'
For experiment 2, the figure caption

**Comment by KateRasheed** [comment]
From this statement, it confirms four as the sample size. "The samples at d0 and d49 were collected from the cages and thus **represent the pooled microbiota of four experimental animals**"

Doesn't it?

Cc: @SvetlanaUP

**Comment by SvetlanaUP** [comment]
Yes, a pooled sample (obtained from several samples) we count as 1.

**Comment by annemarie-sharp** [comment]
@KateRasheed @SvetlanaUP yes there was one pooled sample from each of the six cages, hence 6samples. 

Because there were 24mice and 4mice were put in each cage(hence 6cages) 
So each sample represents 4mice/1cage
It's a lot of calculation and that's why I asked if it's not explicitly mentioned if I should omit it and I later omitted for both experiments.

**Comment by SvetlanaUP** [comment]
@KateRasheed @annemarie-sharp "The stool samples were collected from the cages." Here we count number of cages as number of samples not the animals. They took samples from the floor of the cage, not from each animal separately.

**Comment by KateRasheed** [comment]
Sample sizes effected @annemarie-sharp @SvetlanaUP . Thank you.

**Comment by annemarie-sharp** [comment]
Thank you for your understanding @KateRasheed @SvetlanaUP
